README.md

```diff
@@ -8,17 +8,23 @@ Core repository for the platform team 4 is developing
 1. Run `yarn` to fetch dependencies.
 2. Run `yarn run start:dev` to simultaneously run the frontend and backend.
 
+**For more information regarding frontend and backend specific development, view the READMEs in their respective folders:**
+
+- [Frontend](packages/frontend)
+- [Backend](packages/backend)
+
 # Running with Docker
 
 Running with Docker will ensure that users can quickly get environment up and running so they can quickly iterate on work. The application will work regardless of operating system and configuration.
 
 ## Web app
-Ensure you have set up backend env files first. Checkout backend README for more info.  
 
-1. Run `yarn run start:docker` to simultaneously run the frontend and backend in docker containers.  
+Ensure you have set up backend env files first. Checkout backend README for more info.
+
+1. Run `yarn run start:docker` to simultaneously run the frontend and backend in docker containers.
```

Comment:
```suggestion
1. Ensure you have [set up backend env files](./packages/backend/README.md)
2. Run `yarn run start:docker` to simultaneously run the frontend and backend in docker containers.
```
you can link to the appropriate readme file here. also make sure this is numbered.